receiver/telegrafreceiver/README.md

```diff
@@ -0,0 +1,45 @@
+# Telegraf Receiver
```

Comment:
Maube a "limitations" section would be helfpul and a note there that currently only Gauges are supported?

Reply:
Done, PTAL